guides/common/modules/proc_importing-ansible-roles.adoc

```diff
@@ -1,11 +1,11 @@
 [id="importing-Ansible-roles_{context}"]
 
-= Importing Ansible Roles
-
-You can import Ansible roles from the `/etc/ansible/roles` directory on {Project} or on a {SmartProxy} that has Ansible enabled.
-Ensure that the roles that you import are located in the `/etc/ansible/roles` directory on all {SmartProxies} from where you want to use the roles.
+= Importing Ansible Roles and Variables
 
+You can import Ansible roles and variables from the `/etc/ansible/roles` directory on {Project} or on a {SmartProxy} that has Ansible enabled.
```

Comment:
For what it's worth, this location is configurable in the latest version by using `--foreman-proxy-plugin-ansible-roles-path`. The default is equivalent to `--foreman-proxy-plugin-ansible-roles-path /etc/ansible/roles --foreman-proxy-plugin-ansible-roles-path /usr/share/ansible/roles`.

Reply:
I'll change for GA.

Reply:
Yes, that's fine since this needs to be cherry picked. Just wanted to share the info.